### DISCUSSION TITLE
How to pass status of subdag to parent dag ?

### DISCUSSION THREAD

**Discussion by nikhil1699** [apache/airflow#14999] [Q&A]
Hi 
We are working on Airflow 1.10 and we have created a few nested subdags to suit our requirements.
Suppose we have a parent DAG P 
and then we have 3 subdags A, B, C  and each of these subdags have 1 subdag of their own, lets name them  X,Y,Z respectively. Its not most beautiful design, but it suits our requirement.

Now what we want is to be able to send SUCCESS or FAILURE status of each subdag in Logs of the main parent DAG P. So suppose X fails, and all other are success, I want the information that subdag X failed, in logs for subdag P.

Another thing to mention is that P will SUCCEED even if X fails, I just want the status of X in logs of P.

Thanks alot for your time.

**Comment by turbaszek** [comment]
I'm not sure if this is possible out of the box. 

> in Logs of the main parent DAG P

What do you mean by logs of DAG? I though we have only logs per task not per DAG?

**Reply by nikhil1699** [reply]
By logs of Parent DAG I meant, the logs of all tasks in the parent dag.
So if there's a subdag operator in a parent dag, I wanted that the status of subdag should be loggable in parent dag in some way.
And if its not possible out of the box, can you please suggest some ways to achieve our goal
